app/queries/admin/gdpr_delete_requests_query.rb

```diff
@@ -0,0 +1,19 @@
+module Admin
```

Comment:
My heart really wants a few unit tests for this.  The Cypress test covers the UI and integration, but those are more expensive to run compared to a unit test.  Further the unit tests let us more easily bombard the query.  Some scenarios that aren't covered in the Cypress test involve:

1. Filter on role
2. Multiple people matches where one match is because of username and the other because of email
3. We filter on a role and search on a username, there's a person with the username but not the role

In cypress constructing the necessary data is quite cumbersome; whereas in an RSpec test, FactoryBot makes this a lot easier.

This is an "almost" blocker for me.  I want to further discuss this, and would be happy to pair if you'd like help crafting these.

Diving further into the why of the unit tests even when Cypress provides some coverage, I think about both refactoring and replacement.